syntax = "proto3";

package kvstore.server;

message AddHint {
  uint32 replica = 1;
  uint32 key = 2;
}

message DropHint {
  uint32 replica = 1;
  uint32 key = 2;
}

message HintMessage {
  oneof msg {
    AddHint add_hint = 1;
    DropHint drop_hint = 2;
  }
}
